### DISCUSSION TITLE
期望应用中心增加「最近流行」的排序

### DISCUSSION THREAD

**Discussion by intoyang** [halo-dev/halo#4836] [Ideas]
### 你当前使用的版本

2.9

### 描述一下此特性

如下，在应用中心新增「最近流行」排序，考虑按最近 30 天内下载量进行降序展示。
<img width="903" alt="Screenshot 2023-11-08 at 14 26 53" src="https://github.com/halo-dev/halo/assets/73098982/d96dfb1f-02f2-41d2-a770-430b27ebd87a">

### 附加信息

kind/improvement

**Comment by ruibaby** [comment]
不属于 Halo 本身的 issue，我将转移到 Disscussion 的 Ideas 分组。
